### PR TITLE
Adding isPrime to the problems page to highlight pagination

### PR DESCRIPTION
**Pull request by lalaAdasia00**

### CODE REVIEW

File: problems/isPrime.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  assert(isPrime(5) === true);
+  assert(isPrime(24) === false);
+  assert(isPrime(97) === true);
+  alert('Passed all test cases!');
+tests:
+  - assert(isPrime(5) === true);
+  - assert(isPrime(24) === false);
+  - assert(isPrime(97) === true);
```
Comment (ifeomi) [inline problems/isPrime.yaml:26]: can we add some test cases for negative numbers?

File: problems/isPrime.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      return true;
+    } else {
+      for(var x = 2; x < num; x++) {
+        if(num % x === 0) {
+          return false;
+        }
+      }
+    return true;  
+    }
+  }
```
Comment (ifeomi) [inline problems/isPrime.yaml:41]: this breaks on negative numbers and always returns true, but neg numbers can't be prime

File: problems/isPrime.yaml (hunk 1 above)
Comment (lalaAdasia00) [reply problems/isPrime.yaml:26]: sure

File: problems/isPrime.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  assert(isPrime(97) === true);
+  alert('Passed all test cases!');
+tests:
+  - assert(isPrime(5) === true);
+  - assert(isPrime(-5) === false)
+  - assert(isPrime(24) === false);
+  - assert(isPrime(97) === true);
+solution: |
+  function isPrime(num) {
+    if (num===1 || num < 0){
```
Comment (nicholashusin) [inline problems/isPrime.yaml]:
```suggestion
    if (num <= 1){
```

File: problems/isPrime.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  - assert(isPrime(97) === true);
+solution: |
+  function isPrime(num) {
+    if (num <= 1){
+    return false;
+    } else if(num === 2) {
+      return true;
+    } else {
+      for(var x = 2; x < num; x++) {
+        if(num % x === 0) {
```
Comment (ifeomi) [inline problems/isPrime.yaml]:
```suggestion
    } else if (num === 2) {
      return true;
    } else {
      for (var x = 2; x < num; x++) {
        if (num % x === 0) {
```

**Review by ifeomi — APPROVED** [review]
small comment, lgtm

**Review by chronologos — APPROVED** [review]
